### ISSUE TITLE
 Protection against accidental deletion of an event

### ISSUE THREAD

**Issue by ConradReichel** [OpenSlides/openslides-client#4514]
**Description:**
When working on a committee's meeting, the inhibition threshold for accidentally deleting a meeting is no higher than duplicating it.

**Expected behaviour:**
If the "OK" in the deletion-dialog is pressed an additional dialog should open itself and ask if the meeting should really be deleted,, pressing "OK" here should delete the meeting, pressing cancel should not delete the meeting

**Comment by reiterl** [comment]
There is already a prompt "Are you sure you want to delete this meeting?" in the committee-detail view, I opt against building a second prompt.

**Comment by Elblinator** [comment]
Yes and the described problem is that this prompt has the same behaviour as the dublicate option, which is exactly the problem and could be countered by a second prompt

**Comment by reiterl** [comment]
Thomas said, that GH sometimes asks for a extra information to enter in such cases in the first prompt.
E.g. "enter the id of the meeting to confirm that meeting should be delete" I don't have exp with that.
And it would mean to write a new dialog.

**Comment by Elblinator** [comment]
Something should change in the current behaviour, because the two dialogs look and behave identical besides the word "dublicate"/"delete".
![Screenshot_20250108_122032](https://github.com/user-attachments/assets/b57b99c9-25c1-42aa-914d-034af55fce33)


Potential solutions which could be discussed:
A) The second dialog
B) A visual change in the current dialog (e.g. different colour in the title, or different buttons)
C) An input field in the current dialog, where you have to input sth (e.g. the meeting title) if this solution is choosen please consider "wrong" inputs in the concept (faulty input or input which is not the meeting you are trying to delete).

**Comment by ConradReichel** [comment]
Personally, I find the last option very appealing. Personally, it would probably have saved me the almost clumsy task of having to create a backup in parallel in order to be able to restore the event - I personally fell for this identical looking confirmation message 😥
